clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/customtemplates/CustomTemplateContext.kt

```diff
@@ -240,12 +259,34 @@ sealed class CustomTemplateContext private constructor(
         return argumentValues[name] as? T
     }
 
+    override fun toString(): String {
```

Comment:
any specific reason to overirde this method? is it just for logging purpose/some functionality